Configurations/unix-Makefile.tmpl

```diff
@@ -982,13 +982,17 @@ update: generate errors ordinals
 generate: generate_apps generate_crypto_bn generate_crypto_objects \
           generate_crypto_conf generate_crypto_asn1 generate_fuzz_oids
 
-.PHONY: doc-nits cmd-nits
+.PHONY: doc-nits cmd-nits md-nits
 doc-nits: build_generated
 	$(PERL) $(SRCDIR)/util/find-doc-nits -n -l -e
 
 cmd-nits: build_generated apps/openssl
 	$(PERL) $(SRCDIR)/util/find-doc-nits -c
 
+# Needs ruby and "gem install mdl"
```

Comment:
Another variant is to use the snap store: https://snapcraft.io/install/mdl/debian

Reply:
Do you want me to add that comment?

Reply:
Yes.  There are enough installations out there that use general repositories like snap (when there isn't a distribution-specific package) rather than language specific installers to warrant adding that information

Reply:
(**I** would like that information, especially after all the scars from battles with gem, npm and what not)

Reply:
okay, I added more comments that anyone will ever want or need :)